Add bytao7mao to Contributors list

bytao7mao opened the pull request:
Hello Roshanjossey, thanks for this tutorial, i just made my first contributions, i didn't knew beforehow to do it!

bytao7mao committed:
Add bytao7mao to Contributors list

Roshanjossey commented:
@bytao7mao, I'm quite elated about your pull request. I wanna evolve this project to addresses various problems faced by first-time contributors. I'd love to learn about your journey in open source community, the problems, pain points you had etc.
Could you explain how you felt when you went through the tutorial, made a pull request and learned that I merged it?

 We’ve recently added Facebook share and tweet feature to our web app. Could you please go to https://roshanjossey.github.io/first-contributions/#social-share and share your first contribution to open source?  Also, check out projects with easy issues while you’re there.

bytao7mao commented:
@Roshanjossey well, since the tutorial is very well explained i did not had any problems when i made the pull request. 
About me: i have some experience in Java, C#, Python, Javascript, Photoshop so i'll search for the issues and try to contribute !
Yes, i'll check now the facebook link and share 👍 
Thanks for your fast answer! Cheers

bytao7mao commented:
Will be a share link for Linkedin too ?

Roshanjossey commented:
@bytao7mao, I didn't quite get that question. Do you wanna share a link to your linked in profile too?
if yes, go ahead

bytao7mao commented:
@Roshanjossey I was reefering to the page, maybe i can contribute with an similar icon like facebook and twitter but for Linkedin to share.

Roshanjossey commented:
@bytao7mao, Cool go ahead 👍

bytao7mao commented:
Hello again, i don't have experience in React but i will make the pull request.
- I've added so far the following files: Linkedin.jsx, Linkedin.test.js;
- In the icons.js i've added the svg for linkedin;
- In the SocialShare.jsx i've imported/added the Linkedin also;

Please tell me if i have to change or add anything else, i tried to test it opening index.html but it shows me an empty page...
